src/ExEss/Bundle/DoctrineExtensionsBundle/Schema/Trigger.php

```diff
@@ -146,7 +146,6 @@ public function equals(Trigger $trigger): bool
             \preg_replace('/\s+/', '', $this->getStatement()) === \preg_replace('/\s+/', '', $trigger->getStatement())
             && $this->getTiming() === $trigger->getTiming()
             && $this->getOperation() === $trigger->getOperation()
-            // && $this->getTriggerName() === $trigger->getTriggerName();
```

Comment:
that's commented code, I think it was because I said, during our pair programming session that $trigger->triggerName would work, and we forgot to delete the using of methods afterwards

Reply:
my bad, misinterpreted the diff completely wrong!